server/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+require('dotenv').config({ path: '.env' });
+const axios = require('axios');
+const applyMiddlewares = require('./middlewares');
+
+const server = express();
+const PORT = process.env.PORT || 5000;
+
+applyMiddlewares(server);
+
+server.get('/', (req, res) => {
+  res.send('server running');
+});
+
+server.post(':endpoint([\\/\\w\\.-]*)', (req, res) => {
+  const endpoint = process.env.API_BASE_URL + req.params.endpoint;
```

Comment:
API_BASE_URL can be a plain variable inside this file, no need to move it to .env